### ISSUE TITLE
Ability to add different paths for different users. 

### ISSUE THREAD

**Issue by modem7** [sct/overseerr#609]
**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
It would be useful in certain use cases to be able to add a different download path for different users. 

E.g.
Owner and family = /main/media/folder
Guests and friends = /requests/folder/ (or default sonarr/radarr settings)

It could utilise already implemented features of sonarr and radarr paths, and add a "priority" or "override" system where the custom settings take precedence. 

**Describe alternatives you've considered**
None

**Additional context**
This would be useful in cases where you'd prefer content from certain people to go to certain libraries (especially if there's library restrictions in place)

**Comment by TRaSH-** [comment]
Would be nice if this could be implanted like Ombi has,
profile and paths are set with the user and not the service.

![image](https://user-images.githubusercontent.com/6155095/104087762-56b66a00-5262-11eb-93ba-21984ac65b82.png)

**Comment by modem7** [comment]
> Would be nice if this could be implanted like Ombi has,
> profile and paths are set with the user and not the service.
> 
> ![image](https://user-images.githubusercontent.com/6155095/104087762-56b66a00-5262-11eb-93ba-21984ac65b82.png)

Damn dude, I didn't even realise Ombi had this.....Setting that up now lol

**Comment by mouzzampk2014** [comment]
I am using overseerr as a daily driver and this feature is not stopping me at all but as request management service I think this will be great add on and will help lot of us. Thanks

**Comment by TupacDas** [comment]
this feature will be nice, assigning radarr server to a predefined user will be amazing

**Comment by hitmax117** [comment]
THIS is the one thing i'm missing with Overseerr to replace Ombi. UI and general presentation is great! But setting sonarr/radarr paths per users is a must for me. 
(also reporting issue's/problems would be a nice addon as wel)

**Comment by Epirr** [comment]
This is a super important feature for me, it's the one thing that's stopping me from fully moving from Ombi to Overseerr, please don't close it :c

**Comment by smailpouri** [comment]
Yes, that is one of the main reason I can’t switch to Overseerr. 

Separate Profile and separate path for each user. 
Ombi’s implementation is really well done.

**Comment by ionred** [comment]
+1 for this. @sct  is there any update on this potential implementation?

**Comment by sct** [comment]
> +1 for this. @sct  is there any update on this potential implementation?

Sorry we don't give ETAs. We will get to it when we can!

**Comment by geist325** [comment]
Agreed, SUPER useful feature. I can't completely jump off of Ombi without such a feature. A few of my users have their own root directories, so the garbage that they request isn't fouling up my pristine main library! Overseerr is WAY more polished and simple to use. Fingers crossed for this one.

**Comment by ionred** [comment]
@geist325 exact use case for me too. Couple of users that download pure garbage. Only willing to tolerate it because it’s hidden in a root folder exclusively for them so I’ve got them as my last users on Ombi. Hoping they can action on profile level request handling soon! 🤞🤞🤞

**Comment by landski** [comment]
Wanted to bump this one as I have been crossing my fingers for 2 years, hoping this feature would get implemented.

**Comment by geist325** [comment]
> Wanted to bump this one as I have been crossing my fingers for 2 years, hoping this feature would get implemented.

+1!

**Comment by CarlesLlobet** [comment]
+1 !!!!!

Overseerr is much better in so many ways than Ombi, but it just lacks this critical feature.
Some colleagues have already migrated to Ombi specifically because of this, but I am reluctant to do so (yet).

In my head this should not be that crazy to implement, is there any ETA or info on whether this is something being worked on?

**Comment by caddish** [comment]
+1

**Comment by Made1908** [comment]
+1

**Comment by geist325** [comment]
+1

**Comment by 0verdrive** [comment]
+1.  I thought I could set up different "Services" to the same Radarr/Sonarr server, but with different Root Folders.  But there doesn't appear to be any way to associate a user with a service.

**Comment by JordanRichard** [comment]
Another +1 to this. Would love to see this feature added. Is anyone currently working on a PR for this?

**Comment by retepz** [comment]
I've been wanting to work on this for a while.  A while back I pulled down the code but for the life of me couldnt even get it running much less figure out where to put it.  Maybe sometime during the holidays I'll take a look but don't hold your breath.

**Comment by albertusQuelle** [comment]
Can we use a Tag and tell radarr and qbitorrent to use separate folders?

**Comment by UberPlexCa** [comment]
> Can we use a Tag and tell radarr and qbitorrent to use separate folders?

That may actually work.

**Comment by albertusQuelle** [comment]
Any idea how to realise it?Am 23.12.2024 um 04:21 schrieb JaymanCanada ***@***.***>:﻿

Can we use a Tag and tell radarr and qbitorrent to use separate folders?

That may actually work.

**Comment by norimicry** [comment]
+1, only reason I'd use Ombi, for the better user library management. Any update?

**Comment by shodeira** [comment]
+1

**Comment by Alfy1080** [comment]
+1

**Comment by kaijikan** [comment]
+1 this gets my vote too, this would be a really useful feature!

**Comment by Neil-1980** [comment]
+1

**Comment by ionred** [comment]
All, if this is an absolute must for you, there's a similar feature available in the fork project Jellyseerr (which despite originally targeting Jellyfin does support Plex too). 

The feature is called override rules. So you set a default root folder (or quality or tag) and you can set a rule for specific user(s) to change any of those three values.

The rules also support other criteria, like genre, keyword, or language.

**Comment by kaijikan** [comment]
> All, if this is an absolute must for you, there's a similar feature available in the fork project Jellyseerr (which despite originally targeting Jellyfin does support Plex too).
> 
> The feature is called override rules. So you set a default root folder (or quality or tag) and you can set a rule for specific user(s) to change any of those three values.
> 
> The rules also support other criteria, like genre, keyword, or language.

This is really great to know. I appreciate you spending the time to share the info!

**Comment by geist325** [comment]
> All, if this is an absolute must for you, there's a similar feature available in the fork project Jellyseerr (which despite originally targeting Jellyfin does support Plex too).
> 
> The feature is called override rules. So you set a default root folder (or quality or tag) and you can set a rule for specific user(s) to change any of those three values.
> 
> The rules also support other criteria, like genre, keyword, or language.

While that is a nice workaround, that is what it is. It's not a feature or a fix, just saying.
If a native in-application feature could be implemented it would be a much more elegant and cleaner solution than relying on external tags.

**Comment by ionred** [comment]
It's not a workaround using tags or anything. (Tags were an option for something to override, not a requirement to use this feature) Jellyseerr is a fork of this project that is regularly adding more features to the Overseerr code base. The developers of it also contribute back to Overseerr as well on occasion. 

The feature I mentioned fully addresses the need in the original issue thread comment. While it does not implement it exactly as the original issue requester wished (doing so in the user profile), it solves the folder assignment issue without workarounds in a few clicks. 

Until extremely recently, development on Overseerr had stopped almost completely except for some bug fixes due to life issues for the maintainers, which is why the fork was created. If you're looking for more features in Overseerr and more frequent development it's the way to go. There's also a migration script floating around that lets you convert from OS to JS seamlessly.

**Comment by geist325** [comment]
I just spun up a Jellyseerr docker instance. 
I've imported my users. 
I do not see an option to manage tags per user.
<img width="1017" height="1105" alt="Image" src="https://github.com/user-attachments/assets/e5da823a-65fc-48a6-a941-6448cceba805" />

I see how there's a way to potentially do this in "Services" by managing tags there, but that would require setting up a separate "Service" for each user, which is less than ideal. Maybe I'm missing something.

**Comment by ionred** [comment]
Two things:

1) The original thread was requesting something regarding root folders, primarily. Using the override rules on the services page lets you do that. Set the services radarr/sonarr to default to the folder you'd want your users to fall into, and then set an override for yourself to go to the separate folder for your specific requests. You can specify more than one user in a rule, so in this example if you wanted a spouse's/partner's in your folder too, they can be in the one rule with you. 

2) If you were wanting a specific tags per user, there's an option for that already built in too! Go to the radarr or sonarr instance's specific  settings menu. At the bottom is the option to agree a specific tag for each user's requests.

 ![image](https://github.com/user-attachments/assets/d5036273-c7b4-41e8-a559-a555bd2153c9)

Hope this helps. If you have questions, feel free to ping me on Discord. Oddelder / OddElder#4611. There's also an active Jellyseerr discord too.

**Comment by modem7** [comment]
> Two things:
> 
> 1. The original thread was requesting something regarding root folders, primarily. Using the override rules on the services page lets you do that. Set the services radarr/sonarr to default to the folder you'd want your users to fall into, and then set an override for yourself to go to the separate folder for your specific requests. You can specify more than one user in a rule, so in this example if you wanted a spouse's/partner's in your folder too, they can be in the one rule with you.

Where are you seeing the override settings? Not seeing that on the latest overseer commit (or is this for JS)?

<img width="779" height="866" alt="Image" src="https://github.com/user-attachments/assets/9367f419-22eb-4f1b-8cca-e0953ebda02d" />

**Comment by geist325** [comment]
I think in a perfect world everyone would like something as simple as setting a root folder for that user in the user's settings, much like Ombi. This seems like it will work though. Most people aren't managing more than 2-3 users with their own root folders.

**Comment by ionred** [comment]
> > Two things:
> > 
> > 1. The original thread was requesting something regarding root folders, primarily. Using the override rules on the services page lets you do that. Set the services radarr/sonarr to default to the folder you'd want your users to fall into, and then set an override for yourself to go to the separate folder for your specific requests. You can specify more than one user in a rule, so in this example if you wanted a spouse's/partner's in your folder too, they can be in the one rule with you.
> 
> Where are you seeing the override settings? Not seeing that on the latest overseer commit (or is this for JS)?
> 
> <img width="779" height="866" alt="Image" src="https://github.com/user-attachments/assets/9367f419-22eb-4f1b-8cca-e0953ebda02d" />

@modem7 See my previous message that I was responding to Geist about. There's a fork of Overseerr called Jellyseerr that meets the original requirements of this issue.

**Comment by ionred** [comment]
> I think in a perfect world everyone would like something as simple as setting a root folder for that user in the user's settings, much like Ombi. This seems like it will work though. Most people aren't managing more than 2-3 users with their own root folders. 

I agree it's a decent alternative to set it per user folder. But since the Jellyseerr devs implemented it this way I doubt it will change. But overall, I can route all my media (and that of my 20+ users into a separate folder) in less than 90 seconds, so doesn't matter much.

**Comment by mikemathewson** [comment]
For anyone still struggling with this I have come to the conclusion that it's just easier to maintain one library and assign access via labels in Plex. I use Overseer's ability to assign tags based on the username of the requester and combine that with Kometa to sync the tags over to Plex labels. This also solves the issue of being able to give access to specific media to multiple users without having to duplicate the media or mess around with symlinks.

**Comment by ZiIlaGit104** [comment]
I 2nd this and would add, default/limited quality/path per user option would be amazing!
